package.json

```diff
@@ -41,7 +41,9 @@
     "test-coverage-once": "bin/config-check.js && better-npm-run jest --coverage",
     "test-once": "bin/config-check.js && better-npm-run jest && npm run lint",
     "version-check": "bin/version-check.js",
-    "webpack-dev-server": "npm run build-locales && better-npm-run webpack-dev-server"
+    "webpack-dev-server": "npm run build-locales && better-npm-run webpack-dev-server",
+    "storybook": "cross-env NODE_PATH=src start-storybook -p 9001 -c stories/setup",
```

Comment:
I could not get this working without NODE_PATH :/ I also saw that it was good idea include this package if we are going to use this...